cpe_workspaceone/resources/cpe_workspaceone.rb

```diff
@@ -71,7 +82,7 @@ def macos_enforce_mdm_profiles
       if enforced_device_ws1_profiles.include?(profile_name)
         execute "Sending #{profile_name} for device installation to Workspace One console" do
           # spaces in path, so we need to convert them with gsub
-          command "#{hubcli_path.gsub(/ /, '\ ')} profiles --install #{profile_id}"
+          command node.hubcli_execute("profiles --install #{profile_id}", execute=false)
```

Comment:
Instead of overloading node.hubcli_execute to do two things, lets split it into two node functions...one that assembles the string for the hubcli command and one that actually executes it. It makes this feel a lot cleaner and without having to tack on `execute=false` everywhere and reduces the confusion about what `node.hubcli_execute(...,execute=false)` does.

`node.hubcli_execute` would call `node.hubcli_cmd` to get us the string we need and then execute it like it does now.

Reply:
Yup that's much better. Adding!